### PR TITLE
UX-968, UX-989, UX-990/Add GKE cloud provider and cluster import features

### PR DESCRIPTION
**Pull request by kevinylee1991**
Screen Shot 2021-06-09 at 3 46 21 PM
Screen Shot 2021-06-09 at 3 46 45 PM
Screen Shot 2021-06-09 at 3 51 24 PM

### CODE REVIEW

File: src/app/plugins/kubernetes/components/infrastructure/cloudProviders/AddCloudProviderCredentialStep.tsx [hunk 1]
```diff
@@ -39,25 +42,33 @@ interface Props {
...
 const testsForAws = [{ name: 'AWS account access', status: null }]
 
 const testsForAzure = [{ name: 'Azure account access', status: null }]
 
+const testsForGoogle = [{ name: 'Google account access', status: null }]
+
 const requiredTests = {
   [CloudProviders.Aws]: testsForAws,
   [CloudProviders.Azure]: testsForAzure,
+  [CloudProviders.GKE]: testsForGoogle,
```
Comment (pallabpain) [inline src/app/plugins/kubernetes/components/infrastructure/cloudProviders/AddCloudProviderCredentialStep.tsx:71]: Can we rename this to `CloudProvider.GCP` instead?

Comment (jruddell) [reply src/app/plugins/kubernetes/components/infrastructure/cloudProviders/AddCloudProviderCredentialStep.tsx:71]: under cloud providers yes we should call this GCP or even just `Google`.

File: src/app/plugins/kubernetes/components/common/CloudProviderCard.tsx [hunk 2]
```diff
@@ -44,13 +44,17 @@ const rootPath = '/ui/images/icon-cloudproviders'
 const icons = {
   [CloudProviders.Aws]: `${rootPath}/cloudaws-default${iconSize}.png`,
   [CloudProviders.Azure]: `${rootPath}/cloudazure-default${iconSize}.png`,
+  [CloudProviders.Google]: `${rootPath}/cloudgoogle-default${iconSize}.png`,
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/common/CloudProviderCard.tsx:47]: dont we have different icons for google vs gke?

File: src/app/plugins/kubernetes/components/infrastructure/cloudProviders/GoogleCloudProviderVerification.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+const GoogleCloudProviderVerification = ({ wizardContext, setWizardContext }: Props) => {
+  const classes = useStyles({})
+
+  const [regions, regionsLoading] = useDataLoader(loadCloudProviderDetails, {
+    cloudProviderId: wizardContext.cloudProviderId,
+  })
+
+  console.log(regions, 'regions')
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/infrastructure/cloudProviders/GoogleCloudProviderVerification.tsx:40]: console.log

File: src/app/plugins/kubernetes/components/infrastructure/cloudProviders/UpdateCloudProviderPage.tsx [hunk 4]
```diff
@@ -62,6 +64,7 @@ const formCpBody = (data) => {
 }
 
 export const UpdateCloudProviderForm = ({ onComplete, initialValues }) => {
+  console.log(initialValues, 'initialValues')
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/infrastructure/cloudProviders/UpdateCloudProviderPage.tsx:67]: console.log

File: src/app/plugins/kubernetes/components/infrastructure/cloudProviders/selectors.ts [hunk 5]
```diff
@@ -14,6 +14,7 @@ import { calculateNodeUsages } from '../common/helpers'
 export const cloudProviderTypes = {
   aws: 'AWS',
   azure: 'Azure',
+  gke: 'Google',
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/infrastructure/cloudProviders/selectors.ts:17]: why is gke a cloud provider type? isn't it a cluster type?

File: src/app/plugins/kubernetes/components/infrastructure/clusters/import/actions.ts [hunk 6]
```diff
@@ -21,6 +21,7 @@ export const discoverExternalClusters = async ({
   }
 
   const data = await qbert.discoverExternalClusters(body)
+  console.log(data, 'cluster data')
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/infrastructure/clusters/import/actions.ts:24]: console.log

**Review by jruddell — CHANGES_REQUESTED** [review]
lets fix the enums, GKE isn't a cloud provider its a type of cluster that is imported from google. few other comments / some minor cleanup

File: src/app/plugins/kubernetes/components/common/CloudProviderCard.tsx (hunk 2 above)
Comment (kevinylee1991) [reply src/app/plugins/kubernetes/components/common/CloudProviderCard.tsx:47]: nope, just the one

**Review by jruddell — APPROVED** [review]

**Pull request merged**